_observing-your-data/workload-management/workload-management-dashboard.md

```diff
@@ -7,179 +7,136 @@ nav_order: 60
+## Monitor workload usage
 
-- **Total workload groups**: Displays the total number of workload groups currently configured.
-- **Total groups exceeding limits**: Shows the number of groups currently exceeding their defined CPU or memory thresholds.
-- **Search bar**: Filter workload groups by name.
-- **Create workload group**: Navigates to the page to define a new workload group.
-- **Refresh button**: Updates statistics to reflect the most recent data.
+Use the **Overview** page to monitor workload resource usage across the cluster.
 
-![WLM Overview Interface]({{site.url}}{{site.baseurl}}/images/Workload-Management/Overview.png)
+You can:
 
-### Workload group table
+- Identify how many workload groups are defined and how many are exceeding thresholds.
+- Filter the table by workload group name.
+- View real-time CPU and memory usage for each group using box plots.
+- Navigate to workload group details or top queries in **Query Insights**.
+  - > **Note:** The link is currently unavailable and will be supported starting in OpenSearch Dashboards version 3.3.
 
-Below the summary cards, a table lists all workload groups with key metrics:
+### Interpreting the table
 
-| Column                   | Description                                                               |
-|--------------------------|---------------------------------------------------------------------------|
-| **Workload group name**  | Clickable link to view details for each workload group.                   |
-| **CPU usage**            | Current CPU usage percentage for the group, visualized with a box plot.   |
-| **Memory usage**         | Current memory usage percentage, also shown with a box plot.              |
-| **Total completions**    | The number of queries completed within the group.                         |
-| **Total rejections**     | The number of queries rejected due to resource limits.                    |
-| **Total cancellations**  | The number of queries canceled.                                           |
-| **Top N Queries**        | A link to view the top queries for this workload group in Query Insights. |
+Each row in the table shows key metrics for a workload group:
 
-The **CPU usage** and **Memory usage** columns display resource consumption using a box plot, which provides a visual summary of usage across nodes in the cluster. The plot includes:
+| Column                   | Description                                                                                                                                                            |
+|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Workload group name**  | Clickable link to view group details.                                                                                                                                  |
```

Comment:
> Clickable link to view group details.
It should mention that the detailed page is a granular node level view of the workload group